migration/process-migration/src/main/java/io/camunda/migration/process/adapter/es/ElasticsearchAdapter.java

```diff
@@ -90,10 +89,13 @@ public List<ProcessEntity> nextBatch(final String lastMigratedEntity) throws Mig
                 q ->
                     q.range(
                         m ->
-                            m.field(PROCESS_DEFINITION_KEY)
-                                .gt(
-                                    JsonData.of(
-                                        lastMigratedEntity == null ? "" : lastMigratedEntity))))
+                            m.number(
```

Comment:
❓ Should this be `term()`? Because the process definition key is stored as a keyword in ES.